packages/web-components/src/components/va-file-input/va-file-input.tsx

```diff
@@ -165,7 +165,7 @@ export class VaFileInput {
     this.uploadStatus = 'idle';
     this.internalError = null;
     if (notifyParent) {
-      this.vaChange.emit({ files: [this.file] });
+      this.vaChange.emit({ files: [] });
```

Comment:
I don't know the use case for bubbling up the file info that we're deleting, but currently the multiple file input is expecting this to be empty to mean a file was deleted. We could pass another prop to indicate deletion if we don't like this solution.